### ISSUE TITLE
Core dump version '0x7000f000' is not supported!

### ISSUE THREAD

**Issue by wajihalam** [MathieuDeprez/ESP32_CoreDump_Arduino_1.0.6#3]
Hi Mathieu, 

Great work on the coredump. I am trying to make it work with ESP-IDF v3.3. 

After running the code, I am able to generate the hex which I convert to b64 format. I then run the espcoredump.py file. But, I am getting an error: Core dump version '0x7000f000' is not supported!.

Any suggestions, why I might be getting such an error? I have attached the core.txt file which contains b64 converted codes as well for your reference. 

[core.txt](https://github.com/MathieuDeprez/ESP32_CoreDump_Arduino_1.0.6/files/7262440/core.txt)

**Comment by wajihalam** [comment]
[core.txt](https://github.com/MathieuDeprez/ESP32_CoreDump_Arduino_1.0.6/files/7262439/core.txt)

This is the core.dat file containing the b64 converted codes.

**Comment by MathieuDeprez** [comment]
Hi Wajihalam,

Thank you for that.
I tried to run the "espcoredump.py" command with your "core.txt" (even if I don't have your firmware.elf file) and I got the same kind of error as you : `A fatal error occurred: Core dump version '1879109632' is not supported! Should be up to '1'.`

I think the problem is that your `core.txt` is not correct. I usually use this website: [hexToBase64](https://base64.guru/converter/encode/hex)
Are you sure to copy the whole hex dump?
